src/Sulu/Bundle/AdminBundle/Resources/js/components/Schedules/WeeklySchedule.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import React from 'react';
+import {transformDateToTimeString, transformTimeStringToDate} from '../../utils/Date';
+import {translate} from '../../utils/Translator';
+import DatePicker from '../DatePicker';
+import MultiSelect from '../MultiSelect';
+import Form from '../Form';
+import type {Weekday, WeeklyScheduleEntry} from './types';
+
+type Props = {|
+    index: number,
+    onChange: (index: number, value: WeeklyScheduleEntry) => void,
+    value: WeeklyScheduleEntry,
+|};
+
+export default class WeeklySchedule extends React.Component<Props> {
+    handleDaysChange = (days: Array<Weekday>) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, days});
+    };
+
+    handleStartChange = (start: ?Date) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, start: start ? transformDateToTimeString(start) : undefined});
+    };
+
+    handleEndChange = (end: ?Date) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, end: end ? transformDateToTimeString(end) : undefined});
+    };
+
+    render() {
+        const {value} = this.props;
+
+        return (
+            <Form>
+                <Form.Field colSpan={6} label={translate('sulu_admin.weekdays')} spaceAfter={6}>
```

Comment:
i know the `colSpan` is like this in the design, but dont you think it would be useful to use `colSpan={12}` to see as many days as possible if the select is closed?